thankyou_letters/migrations/10.0.1.0.3/post-migration.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2017 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Monzione Marco
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate(use_env=True)
```

Comment:
When you don't use the environment in the migration, you don't need this. Remove and instead of env parameter, you will get cr directly.